Art/maidxn/styles.css

```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script&family=Handjet&display=swap');
+
+html {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    min-height: 100vh;
+    background: -webkit-gradient(linear, left top, left bottom, from(#000428), to(#004e92)) fixed;
+    font-family: 'Dancing Script', cursive;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    flex-wrap: wrap;
+}
+
+/* .flex-container {
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+} */
```

Comment:
Please don't commit outcommented code.

Reply:
Thank you for the review, I adjusted it like you said, if there are still any problems, please just tell me to fix it. Thanks!